src/app/src/main/java/com/example/bookmark/BorrowedActivity.java

```diff
@@ -1,18 +1,99 @@
 package com.example.bookmark;
 
+import android.content.Intent;
+import android.os.Bundle;
+import android.os.Parcelable;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ListView;
+
+import androidx.appcompat.app.ActionBar;
 import androidx.appcompat.app.AppCompatActivity;
 
-import android.os.Bundle;
+import com.example.bookmark.adapters.BookList;
+import com.example.bookmark.models.Book;
+import com.example.bookmark.models.Owner;
+import com.example.bookmark.models.Request;
+
+import java.util.ArrayList;
 
 /**
- * TODO: Description of class.
+ * This activity shows a user a list of books that they are currently
+ * borrowing. They can select a book which takes them to the
+ * BorrowedBookDetailsActivity where they can see the books details and
+ * return the book.
+ * TODO what else do we want here?
+ *
  * @author Ryan Kortbeek.
  */
 public class BorrowedActivity extends AppCompatActivity {
 
+    ArrayList<Book> borrowedBooks;
+    BookList borrowedBooksAdapter;
+    ListView borrowedBooksListView;
+
+    ActionBar borrowedActionBar;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_borrowed);
+
+        borrowedActionBar = getSupportActionBar();
+        assert borrowedActionBar != null;
+        borrowedActionBar.setTitle("Borrowed");
+
+        borrowedBooksListView = findViewById(R.id.borrowed_books_listview);
+
+        borrowedBooks = new ArrayList<>();
```

Comment:
Could optionally move this to where borrowedBooks is first declared to reduce the visual distance.